src/native/external/libunwind.cmake

```diff
@@ -229,7 +230,7 @@ set(libunwind_la_SOURCES_aarch64_common
 # The list of files that go into libunwind:
 set(libunwind_la_SOURCES_aarch64
     ${libunwind_la_SOURCES_aarch64_common}
-    ${libunwind_la_SOURCES_aarch64_os_local}
+    ${libunwind_la_SOURCES_aarch64_os_local} ${libunwind_aarch64_la_SOURCES_os}
```

Comment:
Why is the aarch64/setcontext.S added only for FreeBSD and the getcontext.S for all targets? For other architectures, both are added for all targets.

Reply:
@janvorli, getsetcontext is for x86 arch only and applies to linux and freebsd OS.
FreeBSD aarch64 requirement was added in 1.8 https://github.com/libunwind/libunwind/commit/032abaaac2ded54cdacffbc32da97c1d49def261. Linux aarch64 does not require it. I think we should make it clear by adding a column "OS" to the table in their readme https://github.com/libunwind/libunwind?tab=readme-ov-file#libc-requirements

Reply:
@janvorli, getsetcontext is for x86 arch only and applies to Linux and FreeBSD OS alike.
FreeBSD aarch64 requirement was added in 1.8 https://github.com/libunwind/libunwind/commit/032abaaac2ded54cdacffbc32da97c1d49def261. Linux aarch64 does not require it. I think we should make it clear by adding a column "OS" to the table in their readme https://github.com/libunwind/libunwind?tab=readme-ov-file#libc-requirements

Reply:
Ah, thank you for the details, that makes sense.